src/OpenTelemetry.Exporter.Stackdriver/OpenTelemetry.Exporter.Stackdriver.csproj

```diff
@@ -1,26 +1,28 @@
 <Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
     <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
-    <TargetFrameworks>net8.0;net6.0;net462</TargetFrameworks>
-    <Description>Stackdriver .NET Exporter for OpenTelemetry.</Description>
+    <TargetFrameworks>net8.0;$(NetStandardMinimumSupportedVersion);net462</TargetFrameworks>
+    <Description>An OpenTelemetry .NET exporter that sends telemetry to Google Stackdriver.</Description>
     <PackageTags>$(PackageTags);Stackdriver;Google;GCP;distributed-tracing</PackageTags>
     <MinVerTagPrefix>Exporter.Stackdriver-</MinVerTagPrefix>
   </PropertyGroup>
 
-  <!--Do not run Package Baseline Validation as this package has never released a stable version.
-  Remove this property once we have released a stable version and add PackageValidationBaselineVersion property.-->
+  <!-- Do not run Package Baseline Validation as this package has never released a stable version.
+  Remove this property once we have released a stable version and add PackageValidationBaselineVersion property. -->
   <PropertyGroup>
     <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Google.Cloud.Monitoring.V3" Version="[3.4.0,4.0)" />
-    <PackageReference Include="Google.Cloud.Trace.V2" Version="[3.3.0,4.0)" />
+    <PackageReference Include="Google.Cloud.Monitoring.V3" Version="[3.8.0,4.0)" />
+    <PackageReference Include="Google.Cloud.Trace.V2" Version="[3.5.0,4.0)" />
```

Comment:
These changes should be included in changelog.
BTW what is the reason to do this? We should support lowest possible version without any security issues.

Reply:
Current used versions are targeting .NET Standard 2.1. The build produced some warnings like

```
warning NU1701: Package 'Google.Api.Gax 4.4.0' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8, .NETFramework,Version=v4.8.1' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
```